siphon/simplewebservice/igra2.py

```diff
@@ -0,0 +1,427 @@
+# Copyright (c) 2013-2015 University Corporation for Atmospheric Research/Unidata.
+# Distributed under the terms of the MIT License.
+# SPDX-License-Identifier: MIT
+"""Read upper air data from the Integrated Global Radiosonde Archive version 2."""
+
+import datetime
+from io import BytesIO
+from io import StringIO
+import itertools
+import sys
+from urllib.request import urlopen
+import warnings
+from zipfile import ZipFile
+
+import numpy as np
+import pandas as pd
+
+from .._tools import get_wind_components
+
+warnings.filterwarnings('ignore', 'Pandas doesn\'t allow columns to be created', UserWarning)
+
+
+class IGRAUpperAir:
+    """Download and parse data from NCEI's Integrated Radiosonde Archive version 2.
```

Comment:
Since this is just a one liner, can you move the closing triple `"""` up to the end of the line? Also add a blank line after for the bot.